### ISSUE TITLE
accessibility tools not accessible

### ISSUE THREAD

**Issue by gioras** [joomla/joomla-cms#40103]
### Steps to reproduce the issue
cannot navigate to the accessibilty tool with keboard navigation.

the tool can be opened with a shortcut but once it open you cannot navigate with arrows.

### Expected result

arrow down or tab shoul navigate between the tool options

**Comment by Quy** [comment]
Duplicate #37210?

**Comment by drmenzelit** [comment]
Thank you @gioras for bringing this up. As @Quy commented above the issue has been already reported, you can follow the discussion there.
I'm closing this as duplicate.
